### ISSUE TITLE
cutfile ignoring data format property Header

### ISSUE THREAD

**Issue by ropable** [roskakori/cutplace#93]
Running cutplace 0.84, I'm trying to undertake basic validation of a CSV file.

  * [CID](https://github.com/ropable/cutplace_issue/blob/master/cid_lpr.csv)
  * [CSV data](https://github.com/ropable/cutplace_issue/blob/master/lpr.csv)

Upon trying the validate the data, I get the following error:

    $ cutplace cid_lpr.csv lpr.csv 
    INFO:cutplace:read CID from "cid_lpr.csv"
    INFO:cutplace:validate "lpr.csv"
    ERROR:cutplace:  lpr.csv (R1C2): cannot accept field 'cost_centre_no': value must be an integer number: 'Cost Centre'

The library appears to be trying to validate the first row in the CSV, despite setting the data format Header = 1 in the CID. The CID appears to be a valid, and any suggestions/assistance you can provide would be appreciated.

**Comment by roskakori** [comment]
Thanks for pointing this out, it's fixed with version 0.8.5, available form PyPI.

The validation of your example still fails but this seems to be caused by broken data:

```
INFO:cutplace:read CID from "cid_lpr.csv"
INFO:cutplace:validate "lpr.csv"
ERROR:cutplace:  lpr.csv (R47C1): values for ['serial_no'] must be unique: ('DEC-2JPK92S.',) (see also: lpr.csv (R42C1): location of first occurrence)
```

**Comment by ropable** [comment]
Yes, the second validation failure was one that I was actually trying to induce during validation. Thanks for the fix!
